packages/deployments/contracts/contracts/messaging/connectors/polygon/lib/Merkle.sol

```diff
@@ -17,7 +17,7 @@ library Merkle {
 
     bytes32 proofElement;
     bytes32 computedHash = leaf;
-    for (uint256 i = 32; i <= proof.length; i += 32) {
+    for (uint256 i = 32; i < proof.length + 1; i += 32) {
```

Comment:
`proof.length` is not a constant. This change does not necessarily save gas. 
- https://github.com/spearbit-audits/connext-nxtp/issues/98 does not apply.

Reply:
fixed: https://github.com/connext/nxtp/pull/2514/commits/46281c3ac81f3289ab7d94044f22f3b05580ae5b